tests/unit_tests/test_export.cpp

```diff
@@ -90,6 +89,7 @@ namespace {
 
 #include <cpp_bindgen/array_descriptor.h>
 #include <cpp_bindgen/handle.h>
+#include <cpp_bindgen/string_descriptor.h>
 
 #ifdef __cplusplus
 extern "C" {
```

Comment:
Should we add a string_descriptor case here for completeness?

Reply:
done